packages/dynamic-import-vars/package.json

```
@@ -0,0 +1,70 @@
+{
+  "name": "@rollup/plugin-dynamic-import-vars",
+  "version": "1.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Resolving dynamic imports that contain variables.",
+  "license": "MIT",
+  "repository": "rollup/plugins",
+  "author": "LarsDenBakker",
+  "homepage": "https://github.com/rollup/plugins/tree/master/packages/dynamic-import-vars",
+  "bugs": "https://github.com/rollup/plugins/issues",
+  "main": "dist/index.js",
+  "engines": {
+    "node": ">= 8.0.0"
```

```suggestion
    "node": ">= 10.0.0"
```

Node 8 isn't supported by LTS anymore. 🎉